python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1431,11 +1431,12 @@ def convert_mean(node, **kwargs):
     name = node["name"]
     proc_nodes = kwargs["proc_nodes"]
     inputs = node["inputs"]
+    attrs = node["attrs"]
 
     mx_axis = node.get("attrs", {}).get("axis", None)
     axes = convert_string_to_list(str(mx_axis)) if mx_axis is not None else None
 
-    keepdims = int(node.get("attrs", {}).get("keepdims", 0))
+    keepdims = get_boolean_attribute_value(attrs, "keepdims")
```

Comment:
Could you fix this in other places too? convert_max, convert_min etc might have the same problem. Maybe argmin, argmax too.

Reply:
I've done so for all occurrences of `keepdims` in `_op_translations.py`, hopefully without changing any semantics other than fixing this.